pkg/compute/init.go

```diff
@@ -118,6 +117,10 @@ func (c *InitCommand) Exec(in io.Reader, out io.Writer) (err error) {
 			StarterKits: c.Globals.File.StarterKits.AssemblyScript,
 			Toolchain:   NewAssemblyScript(),
 		}),
+		NewLanguage(&LanguageOptions{
+			Name:        "other",
+			DisplayName: "Other ('bring your own' WASM binary)",
```

Comment:
> WebAssembly (abbreviated Wasm)

From https://webassembly.org/ its Wasm not WASM 😉  
```suggestion
			DisplayName: "Other ('bring your own' Wasm binary)",
```

Similar comments elided for the rest of the PR, there are many references, some `WASM` and some `wasm`.

Reply:
Oh that's annoying 🙂 as I could have sworn I Google'd the naming and saw WASM used 🤦🏻 

Thanks for flagging this @phamann 👍🏻